test/commands/UnbanBanCommandTest.ts

```diff
@@ -22,7 +22,7 @@ import { read as configRead } from "../../src/config";
 import { RULE_ROOM, RULE_SERVER, RULE_USER } from "../../src/models/ListRule";
 
 function createTestMjolnir(defaultShortcode: string|null = null): Mjolnir {
-    const config = configRead();
+    const config = configRead(false);
```

Comment:
I'm still not confident in this, since this is really trying to be an option whether to load a default if the "original" wasn't found. But we would need to change this in a few more places now that the appservice exists.